js/serve.json

```diff
@@ -0,0 +1,12 @@
+const handler = require('serve-handler');
```

Comment:
Why this file is needed?
For me everything works without any additional json file.